pkg/virt-controller/watch/vm_test.go

```diff
@@ -3104,6 +3104,21 @@ var _ = Describe("VirtualMachine", func() {
 			Expect(vmi.Annotations).To(Equal(annotations))
 		})
 
+		It("should not copy annotations from vm to vmi", func() {
```

Comment:
The original test tested kubernetes and kubevirt labels. Is it true for all labels?

Reply:
This semantic is not obvious. Can you document in the test name or the commit message the reason we do not copy vm labels to vmi ?

Reply:
I've already added it to the commit message:
```
Additionally, remove irrelevant functional test
that was testing carrying annotations from VM
to VMI and ignoring "kubevirt.io" and
"kubernetes.io" labels. This filtering and
functionality was removed in PR https://github.com/kubevirt/kubevirt/pull/3110.

Instead, add a new unit test that tests the
scenario where VM's annotations can not be
carried to VMI annotations.
```